### ISSUE TITLE
Operators should be namespaced

### ISSUE THREAD

**Issue by hanrelan** [mistyharsh/bs-marble#3]
Since the `Operators` module isn't namespaced, it conflicts with any other modules called Operators in the project (I happen to have one in mine). 
I think it might be more ergonomic if it were called `Rx_Operators` and then a `module Operators = Rx_Operators` was added to the `Rx` module?

**Comment by mistyharsh** [comment]
@hanrelan Yes. They should be. I just kept it like that since namespace **was not** the the default behavior when I started building it. I intend to change it when I resume my work on these bindings.
